### ISSUE TITLE
Support to skip N header rows

### ISSUE THREAD

**Issue by breznak** [ben-strasser/fast-cpp-csv-parser#22]
Hi, 

is there a way to skip arbitrary `N` rows of the header? Our CSV files look like this: 
```
colName1, colName2, ...
meta1,meta2
comment1, comment2
data1-1,data2-1
data1-2,data2-2
....
```
Is there a way to parse this? I'd call `set_header("colName1", "colName2")`; then something like: `skipToRow(4)`, ...read data

If not, would you consider implementing that? 
Thank you,

**Comment by ben-strasser** [comment]
Hi,

does the recent commit that adds CSVReader::next_line help?

There is a mechanism to skip comment lines. However, you need to be able
to tell by only looking at the line whether it is a comment, i.e., a
rule similar to: "The first 4 lines are comments" does not work with
that mechanism because you do not know the line number.

Best Regard
Ben Strasser

**Comment by breznak** [comment]
Works like a charm! :clap: 

This is exactly the functionality we needed. 
Thank you very much Ben. 

(I love open source, took 16 mins from reported bug to fix) :wink: 

Fixed in  646bbe5
